app/controllers/publications_controller.rb

```diff
@@ -9,6 +9,11 @@ def index
     @filter.publications_search
 
     respond_to do |format|
+      @content_item = Whitehall
```

Comment:
sorry, I should have been clearer. I think that in this instance, the `@content_item` initialisation has been pasted into the wrong place!